### ISSUE TITLE
Change sign for nscng in hail/graupel budget

### ISSUE THREAD

**Issue by Katetc** [ESCOMP/PUMAS#19]
There was discussion about this change, and Trade made it in the PPE code, but I think it should be added to the cam/release code as part of an earlier PR.

```
@@ -2145,7 +2205,7 @@ subroutine micro_mg_tend ( &
        nsubs(i,k)=0._r8

        if (do_hail .or. do_graupel) then        
-           dum = ((-nsagg(i,k)-nsubs(i,k)+ngracs(i,k))*precip_frac(i,k)-nprci(i,k)*icldm(i,k)-nscng(i,k)*lcldm(i,k))*deltat
+           dum = ((-nsagg(i,k)-nsubs(i,k)+ngracs(i,k))*precip_frac(i,k)-nprci(i,k)*icldm(i,k)+nscng(i,k)*lcldm(i,k))*deltat
        else
           dum = ((-nsagg(i,k)-nsubs(i,k)-nnuccr(i,k))*precip_frac(i,k)-nprci(i,k)*icldm(i,k))*deltat
        end if
```

**Comment by andrewgettelman** [comment]
Yes, I think that is a bug in my recollection? Maybe @trudeeidhammer can confirm this?

**Comment by trudeeidhammer** [comment]
Yes, the sign in front of nscng was wrong.

**Comment by andrewgettelman** [comment]
This I believe has been corrected by Trude on May 1 2021 in commit: https://github.com/ESCOMP/PUMAS/commit/d2110a0448046f97ed46204baa6adef312d62835

Not sure exactly where that is, but assuming it's done.

**Comment by Katetc** [comment]
Fixed in commit 65804082 , tag pumas_cam-release_v1.13
